include/rfb-proto.h

```diff
@@ -87,6 +89,20 @@ enum rfb_vencrypt_subtype {
 	RFB_VENCRYPT_X509_PLAIN,
 };
 
+enum rfb_resize_initiator {
+	RFB_RESIZE_INITIATOR_SERVER = 0,
+	RFB_RESIZE_INITIATOR_THIS_CLIENT = 1,
+	RFB_RESIZE_INITIATOR_OTHER_CLIENT = 2,
+};
+
+enum rfb_resize_status {
+	RFB_RESIZE_STATUS_NO_ERROR = 0,
```

Comment:
Nit: `NO_ERROR` seems an odd name for a "success" status. Maybe just call this `SUCCESS` instead?

Reply:
Renamed to `SUCCESS` now. I had taken the text from the table in https://github.com/rfbproto/rfbproto/blob/master/rfbproto.rst#extendeddesktopsize-pseudo-encoding.